docs/CHANGELOG.rst

```diff
@@ -5,7 +5,6 @@ Change Log
 All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
-
```

Comment:
Don't remove this line. Is supposedly there on purpose ;)

Reply:
Done.